schematic/utils/csv_utils.py

```
@@ -187,7 +189,7 @@ def check_schema_definition(schema_definition: pd.DataFrame) -> bool:
         "Requires" in list(schema_definition.columns) or
         "Requires Component" in list(schema_definition.columns)
     ):
-        print(
+        logger.error(
             "The input CSV schema file contains the 'Requires' and/or the 'Requires "
             "Component' column headers. These columns were renamed to 'DependsOn' and "
             "'DependsOn Component', respectively. Switch to the new column names."
```

`raise Exception` instead of using `logger.error()`.